### ISSUE TITLE
Fatal Error Reported Using grunt-sass Version 1.1.0

### ISSUE THREAD

**Issue by tlattimore** [sass-eyeglass/eyeglass#69]
When following [Grunt integration](https://github.com/sass-eyeglass/eyeglass/blob/master/docs/integrations/grunt.md) I am getting this error reported:

```
Fatal error: Arguments to path.resolve must be strings
```

### Steps to reproduce:

Add the following to your `package.json`

```javascript
{
  "devDependencies": {
    "eyeglass": "0.6.3",
    "grunt": "0.4.5",
    "grunt-sass": "1.1.0"
  }
}
```

Create a file named `Gruntfile.js` that contains this: 

```javascript
'use strict';

var eyeglass = require("eyeglass");

module.exports = function(grunt) {
    grunt.initConfig({
        watch: {
            sass: {
                files: ['*.{scss,sass}', 'Gruntfile.js'],
                tasks: ['sass:dist']
            },
        },
    sass: {
      options: require("eyeglass").decorate({
        sourceMap: true
      }),
      dist: {
        files: {
          'style.css': 'style.scss'
        }
      }
    }
    });
    grunt.registerTask('default', ['sass:dist']);
    grunt.loadNpmTasks('grunt-sass');
};

```
Create a file named `style.scss` with the following: 

```css
.title {
    content: 'Eyeglass is Awesome!';
}
```

Than run:
```bash
grunt
```

**Comment by eoneill** [comment]
What versions of things are you using?

I was able to successfully run your code with the following:

```
eyeglass@0.6.3
grunt@0.4.5
grunt-contrib-watch@0.6.1
grunt-sass@1.0.0
node-sass@3.3.3
```

**Comment by tlattimore** [comment]
Sorry, @eoneill. I should have my `package.json` to this. 

I was running `grunt-sass` version `1.1.0` but when I roll back to `1.0.0` I can no longer replicate this error and eyeglass appears to be working correctly. I will update the issue description to reflect this. 

I am not sure if this should be left open, or whether it is an issue that should be filed against `grunt-sass`?

**Comment by eoneill** [comment]
Ah, I missed that part in your initial report. The issue is probably that `grunt-sass@1.1.0` is using `node-sass@^3.4.0`. Unfortunately, `3.4.0` (and `3.4.1`) contain a breaking change that prevents eyeglass from working correctly: https://github.com/sass/node-sass/pull/1220

The good news, this should be fixed in `node-sass@3.4.2` in the next few days. Until then, use `grunt-sass@1.0.0`. I'll keep this ticket open until `3.4.2` lands, at which point `grunt-sass@1.1.0` should work.

**Comment by tlattimore** [comment]
Thanks for clarifying this, @eoneill! :+1:

**Comment by eoneill** [comment]
resolved with `node-sass@3.4.2` and `eyeglass@0.6.5`
